ksql-functional-tests/src/test/resources/query-validation-tests/literals.json

```diff
@@ -0,0 +1,102 @@
+{
+  "tests": [
```

Comment:
I think we should also add a case for a 0-prefixed literal (e.g. 0123 should be treated as 123)